### PR TITLE
reset epoch for mainnet26

### PR DESCRIPTION
**Pull request by vishalchangrani**

### CODE REVIEW

**Review by joshuahannan — COMMENTED** [review]
I don't know what these argument values need to be, so someone else will need to create them

**Comment by zhangchiqing** [comment]
```
[{"type":"UInt64","value":"146"},{"type":"String","value":"f7b85cb52158c72cf86d88c695489986"},{"type":"UInt64","value":"0"},{"type":"UInt64","value":"719999"},{"type":"UInt64","value":"755999"}]
```

File: transactions/reset-epoch/2024/sept-25/arguments.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "type":"UInt64",
```
Comment (vishalchangrani) [inline transactions/reset-epoch/2024/sept-25/arguments.json:3]: updated args as per https://github.com/onflow/service-account/pull/330#issuecomment-2374663412 (these args were derived from the output of the network upgrade steps on the ansible box)

**Review by joshuahannan — APPROVED** [review]

**Comment by joshuahannan** [comment]
@vishalchangrani Can we merge this?

**Pull request merged**